src/serialize.rs

```diff
@@ -21,7 +21,7 @@ use std::{fmt, io, rc::Rc, str::FromStr};
 
 /// Available export formats.
 // If you add or remove variants, remember to update the CLI docs in `src/bin/nickel.rs'
-#[derive(Copy, Clone, Eq, PartialEq, Debug, Default)]
+#[derive(Copy, Clone, Eq, PartialEq, Debug, Default, clap::ValueEnum)]
```

Comment:
I wonder, do we still need the `impl FromStr for ExportFormat`, if we now delegate parsing to clap? Or maybe we are parsing `ExportFormat` values outside of clap?

Reply:
Deriving `ValueEnum` implements `FromStr` for the type, so the manual implementation isn't needed even if you are using `ExportFormat`s outside of clap. The only exception would be if you want to do something more complex inside the manual `FromStr` implementation. However, the code does not so removing the manual implementation is perfectly fine.

Fixed in ded632fb